ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -490,3 +490,19 @@ def getUserDict(username):
     if value:
       resDict[key.replace(relPath, '')] = value
   return S_OK(resDict)
+
+
+def getEmailsForGroup(groupName):
+  """ Get email list of users in group
+
+      :param basestring groupName: DIRAC group name
+
+      :return: list
+  """
+  emails = []
+  for username in getUsersInGroup(groupName, defaultValue=[]):
+    email = gConfig.getValue("%s/Users/%s/Email" % (gBaseRegistrySection, username), [])
+    if not isinstance(email, list):
```

Comment:
shouldn't the `getValue` call already return a list ?

Reply:
I thought it always returned a string? If so, would it be better to simplify:
```python
     email = gConfig.getValue("%s/Users/%s/Email" % (gBaseRegistrySection, username), '')
     email = email.replace(' ', '').split(',')
```

Reply:
I thought it always returned a string? If so, would it be better to simplify:
```python
     email = gConfig.getValue("%s/Users/%s/Email" % (gBaseRegistrySection, username), '')
     email = email.replace(' ', '').split(',')
```

Reply:
Yes, how I understand returned value type is that set in the last argument. I didn't know about that.

Reply:
Fixed

Reply:
@chrisburr yeah, in `getValue` `defaultValue` is actually `defaultType`... and if `defaultType` is an iterable, it does the split for you...